docs/src/installation.md

```diff
@@ -20,9 +20,22 @@ julia> Pkg.add(url="https://github.com/SpeedyWeather/SpeedyWeather.jl", rev="mai
 ```
 In a similar manner, other branches can be also installed. You can also type, equivalently,
 ```julia
-julia> ] add https://github.com/SpeedyWeather/SpeedyWeather.jl#main
+julia>] add https://github.com/SpeedyWeather/SpeedyWeather.jl#main
```

Comment:
why this space was deleted?
it's really minor, but at least on my repls there is an empty space between `>` and whatever I want to start typing

Reply:
You are right! I just felt wrong to type " " when I don't type it in the REPL but didn't think about it too much. Will correct it now!

Reply:
space is back in with [d2f7133](https://github.com/SpeedyWeather/SpeedyWeather.jl/pull/514/commits/d2f713370ff34715264ef24a8176cb1caa53c28d)

Reply:
this is really my OCD at its best...